data/remnant missions.txt

```diff
@@ -495,3 +565,691 @@ mission "Remnant: Technology Available"
+	name "Remnant bounty"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Salvage 4: offered"
+		random < 60
+	npc kill
+		government "Korath"
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
+	on complete
+		payment 300000
+		dialog "A Remnant military leader thanks you for hunting down the Korath ship <npc>, and gives you the agreed-upon payment of <payment>."
+
+
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to <destination> to see the Quarg."
+	minor
+	source
+		government "Remnant"
+	passengers 3
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		has "Remnant: Salvage 1: done"
+		has "First Contact: Quarg: offered"
+		random < 40
+	on offer
+		conversation
+			`As you relax in the cafeteria on <origin> you are approached by a young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suppose she is a researcher or an engineer.`
+			`	"Hello Captain <last>. My name is Dawn, and I have been spending the past few days skimming the archive you retrieved. I wanted to ask you a few questions. May I?"`
+			`	She slides into the chair opposite you without waiting for your reply, and pulls out a datapad. "Firstly, our histories mention how humanity met a race called the 'Quarg' and had some ongoing dialogs with them, but not much beyond that. Now here in this history that you brought us, it says that humanity has started exchanging technology with them! Is this true?" She looks at you intently. You sign negatively, and she dejectedly continues. "Could you explain about them, then?" You reply that the Quarg have allowed humanity to live near them, and that the Quarg are usually willing to answer questions, including scientific ones. You continue, elaborating that while the Quarg don't seem to mind having an innovative shipyard studying them as best it can, they don't actually share their technology with anyone.`
+			`	After several hours of telling her about meeting the Quarg and visiting their world it finally occurs to you that it would be easy enough to take her to visit the Tarazed system.`
+			choice
+				`	(Offer to take her there.)`
+				`	(Don't offer.)`
+					decline
+			`	 You offer to take her and a couple of her colleagues to <destination>. She eagerly accepts, although she expresses a preference for landing on the human world, so they can observe the Quarg ships from a distance, as they would rather not draw any attention to themselves. She runs out of the cafeteria after telling you they will meet you at your ship in half an hour.`
```

Comment:
This is weird once you subsitute `<destination>` with `Wayfarer`. She expresses her preference  for landing on Wayfarer after being offered to land on Wayfarer?